ChangeLog

```diff
@@ -6,7 +6,7 @@ What's New in astroid 2.7.0?
 ============================
 Release Date: TBA
 
-
+* Added support to infer return type of typing.cast()
```

Comment:
@Pierre-Sassoulas Do you want to include it in `2.6.1`? In that case it would need to be moved to the section below.